packages/project/tsconfig.json

```diff
@@ -1,12 +1,18 @@
-// Extends can't use node's resolution rn https://github.com/dividab/tsconfig-paths/issues/209#issuecomment-1177261091
 {
   "extends": "../../node_modules/@webstudio-is/tsconfig/base.json",
   "include": ["./src/**/*.ts"],
   "compilerOptions": {
+    "composite": true,
```

Comment:
still wanna put it into the reusable config?

Reply:
Done. I don't fully know how it will impact your tsconfig that targets the remix app (since it also extends the base) but I don't think it should affect it negatively. Worst case scenario - we gonna revert this change and put those back into individual configs.